e2e/js_run_devserver/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+
+# Integration test for js_run_devserver run with ibazel
+
+bazel run -- @pnpm//:pnpm --dir "$PWD" install
```

Comment:
thought for later: this would be a nice thing to syntax sugar in the rules_js plugin for aspect-cli